migrations/000001_create_rh_accounts_table.up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS rh_accounts(
```

Comment:
Is there a way we can contain the `up` and `down` in a single file in `migrations/`?

Reply:
No, we can't. This is how [golang -migrate](https://github.com/golang-migrate/migrate) is design to work.